web/lib/firebase/api.ts

```diff
@@ -88,3 +88,7 @@ export function acceptChallenge(params: any) {
 export function getCurrentUser(params: any) {
   return call(getFunctionUrl('getcurrentuser'), 'GET', params)
 }
+
+export function createDashboard(params: any) {
```

Comment:
I really think we should type these params, even though that's not the pattern so far.
We have to manually keep them in sync with the cloud function, but I think it's worth it.